yoeo/utils/utils.py

```diff
@@ -33,20 +36,21 @@ def worker_seed_set(worker_id):
     np.random.seed(ss.generate_state(4))
 
     # random
-    worker_seed = torch.initial_seed() % 2**32
+    worker_seed = torch.initial_seed() % 2 ** 32
     random.seed(worker_seed)
 
 
 def to_cpu(tensor):
     return tensor.detach().cpu()
 
 
-def load_classes(path):
-    """
-    Loads class labels at 'path'
-    """
-    with open(path, "r") as fp:
-        names = fp.read().splitlines()
+def load_classes(path: str) -> dict:
+    with open(path, 'r', encoding="utf-8") as fp:
+        names = yaml.load(fp, Loader=yaml.FullLoader)
```

Comment:
Is the FullLoader needed? The code is not really security critical but we could use a `yaml.safe_load` instead.

Reply:
changed